markdown/events/2022/hackathon-october-2022.md

```diff
@@ -17,4 +17,54 @@ This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barc
 
 In addition to the in-person event, we will endeavour to run a hybrid option online in [gather.town](https://gather.town/).
```

Comment:
Sorry, I know you didn't touch this bit but as you're here anyway.. Maybe can move this into the first paragraph. eg. something like:

```
This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barcelona, Spain 🇪🇸 and online in  [Gather.town](https://gather.town/). Find out more information about the Nextflow Summit, please visit <https://summit.nextflow.io>. 
```